packages/middleware/src/createServer.ts

```diff
@@ -1,11 +1,13 @@
-import consola from "consola";
 import cookieParser from "cookie-parser";
 import cors from "cors";
 import express from "express";
 import type { HelmetOptions } from "helmet";
 import helmet from "helmet";
 import http, { Server } from "node:http";
 import { createTerminus } from "@godaddy/terminus";
+import { LoggerFactory, LoggerType } from "@vue-storefront/logger";
+import { LoggerManager, wrapLogger } from "./loggerManager";
```

Comment:
I am not sure why wrapLogger is exported from the logger manager, loggerManager must export only the class + types eventually, no more than 1 object/class.